data/json/items/gun/545x39.json

```diff
@@ -35,18 +35,18 @@
     "bashing": 11,
     "material": [ "steel", "plastic" ],
     "symbol": "(",
-    "color": "brown",
+    "color": "dark_gray",
     "ammo": "545x39",
     "skill": "rifle",
     "dispersion": 150,
     "recoil": -10,
     "durability": 7,
-    "barrel_length": 1,
```

Comment:
Look at the diff. You moved this field - it used to go before `valid_mod_locations`. The precedent are the files themselves (for linted content) or `tools/format/examples`.

Looking for this sort of thing is also useful in keeping the line count of your PR's low (irrespective of linting). Moving lines around unnecessarily also breaks git history.